WAW.API/Chat/Mapping/ModelToResourceProfile.cs

```diff
@@ -0,0 +1,12 @@
+using AutoMapper;
+using WAW.API.Chat.Domain.Models;
+using WAW.API.Chat.Resources;
+
+namespace WAW.API.Chat.Mapping;
+
+public class ModelToResourceProfile: Profile{
+  public ModelToResourceProfile() {
+    CreateMap<ChatRoom, ChatRoomResource>();
+    CreateMap<Message, MessageResource>();
+  }
+}
```

Comment:
This is not right. Check how mapping profiles are injected in any of the other bounded contexts. For example, this is the Auth model to resource mapping profile (also check the file naming convention):

https://github.com/futureleadersupc/waw-backend-cs/blob/532467fdd326275b3666be65e4aaf3ae0d7a3bfe/WAW.API/Auth/Mapping/AuthModelToResourceProfile.cs#L7-L11

This `Register method is then called in `ModelToResourceProfile` class in the `Shared` bounded context:

https://github.com/futureleadersupc/waw-backend-cs/blob/532467fdd326275b3666be65e4aaf3ae0d7a3bfe/WAW.API/Shared/Mapping/ModelToResourceProfile.cs#L8-L14